dnf-behave-tests/dnf/install-remove.feature

```diff
@@ -311,4 +311,27 @@ Scenario: User-installed packages are not removed as unused dependencies
         | remove-unused | sugar-0:1.0-1.x86_64              |
 
 
+@dnf5daemon
+Scenario: Install with glob
+   When I execute dnf with args "install 'water_*'"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                           |
+        | install       | water_still-0:1.0-1.x86_64        |
+        | install       | water_carbonated-0:1.0-1.x86_64   |
+   When I execute dnf with args "remove 'water_*'"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                           |
+        | remove        | water_still-0:1.0-1.x86_64        |
+        | remove        | water_carbonated-0:1.0-1.x86_64   |
+
+
+@dnf5daemon
```

Comment:
It seems there is no `--advisories=..` option for the `dnf5daemon-client` yet.
It is causing the test to fail.

Reply:
Thanks, I took out the @dnfdaemon tag there.